packages/synapse-react-client/src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -65,7 +65,7 @@ async function addToDownload(
     closeConfirmationFn()
     return [StatusEnum.LOADING_INFO, '']
   } catch (error) {
-    displayToast('danger', undefined, error.reason)
+    displayToast(error.reason, 'danger')
```

Comment:
Parameter order was wrong, so 'danger' was the message and undefined was the message